tests/test_tensorflow_data_api.py

```diff
@@ -146,3 +146,35 @@ def test_dataset_length(self, tmpdir, model):
             )
 
             assert len(tiledb_dataset) == ROWS
+
+    def test_dataset_generator(self, tmpdir, model):
```

Comment:
Maybe clearer naming? You check shapes here.

Reply:
Ok I will fix generally I test generator's output. Shapes is just the way to justify that batching is working correctly.